src/sentry/models/organization.py

```diff
@@ -349,14 +349,18 @@ def has_single_owner(self):
         )
         return len(owners[:2]) == 1
 
-    def get_members_with_org_roles(self, roles: Collection[str]):
-        members_with_role = set(
-            self.member_set.filter(
-                role__in=roles,
-                user__isnull=False,
-                user__is_active=True,
-            ).values_list("id", flat=True)
+    def get_members_with_org_roles(
+        self,
+        roles: Collection[str],
+        include_null_users: Optional[bool] = False,
```

Comment:
The type annotation can just be `bool`. There's no reason to have people pass `None` instead of `False`.